src/App.jsx

```diff
@@ -61,12 +63,23 @@ function App() {
     setTodoList([...filteredTodoList]);
   }
   return (
-    <>
-      <p>{isLoading ? "Loading..." : "" }</p>
-      <h1>Todo List</h1>
-      <AddTodoForm onAddTodo={addTodo}/>
-      <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>
-    </>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={
```

Comment:
What you have is great so far! Here's one thing you could do to take this to the next level
```suggestion
        <Route path="/" element={<ComponentName/>}
```